src/tests/test_journal_vdev.cpp

```diff
@@ -470,6 +504,107 @@ TEST_F(VDevJournalIOTest, Recovery) {
     }
 }
 
+TEST_F(VDevJournalIOTest, MultipleChunkTest) {
```

Comment:
General comment for this test case: if this is a basic test for multiple chunks, fixed chunk-size, data-size is okay. 

However from this PR description, this test case is also going to be used for long hour running, I think we should make chunk-size and data_size configurable and choose a different size every run, so that we can achieve some randomized tests. 

Also we should this test itself seems only did a few insertions each for loop. We should also randomize the number of insertions along with truncations. 

Another category this test didn't cover is the crash recovery, which I think is a must have for journal vdev to make this feature stable.

Reply:
Added two issues to track this.
https://github.com/eBay/HomeStore/issues/379
https://github.com/eBay/HomeStore/issues/378